app/views/catalog/_find_this_item_block.html.erb

```diff
@@ -1,5 +1,9 @@
 <% doc_presenter = show_presenter(document) %>
 <% fields = presenter.new(document: doc_presenter.fields_to_render).terms %>
+<% def purl doc_id 
```

Comment:
Is it possible to define this new method in a helper file instead of a view file?

Reply:
Or maybe in the presenter

Reply:
Will do.